build.gradle

```diff
@@ -3,20 +3,20 @@
 buildscript {
     ext.kotlin_version = '1.3.61'
     ext.versions = [
-            'java': JavaVersion.VERSION_1_8,
-            'androidGradlePlugin': '3.5.3',
-            'googleServices': '3.2.1',
-            'compileSdk': 29,
-            'buildTools': '28.0.3',
-            'minSdk': 16,
-            'targetSdk': 29,
-            'supportLibrary': '26.1.0',
-            'constraintLayout': '1.1.3',
-            'firebase': '10.0.1',
-            'retrofit': '2.0.0-beta4',
-            'okhttp': '3.6.0',
-            'ion': '2.1.8',
-            'videoAndroid': '5.1.1'
+            'java'               : JavaVersion.VERSION_1_8,
+            'androidGradlePlugin': '3.6.1',
+            'googleServices'     : '3.2.1',
+            'compileSdk'         : 29,
+            'buildTools'         : '28.0.3',
+            'minSdk'             : 16,
```

Comment:
Another nit pick change. Can we get rid of the extra space here? This won't be consistent with our formatting for all of our other android projects.

Reply:
I did't manually do this... So something in Android studio must have done that... Let me see if I can get it to revert..

Reply:
It was this commit: https://github.com/twilio/video-quickstart-android/pull/480/commits/edca951ae89967242abb6e2182ed07bc62a5b643

Manually reverting.